jmeter-service/chart/templates/deployment.yaml

```diff
@@ -147,6 +147,16 @@ spec:
                 fieldRef:
                   fieldPath: metadata.labels['app.kubernetes.io/name']
             {{- end }}
+            - name: OAUTH_CLIENT_ID
+              value: "{{ (((.Values.distributor).config).oauth).clientID }}"
+            - name: OAUTH_CLIENT_SECRET
```

Comment:
I think the same applies here as for the bridge: secret values should not be used as env variables. The k8s secret should be mounted as a volume and then read from there...

Reply:
Thanks @mowies for this remark.. I think we agreed with @thisthat and @agrimmer that we will keep it like env vars for now. correct?

Reply:
Correct. Currently, we prefer to consistently use ENV vars.